packages/expo-build-properties/README.md

```diff
@@ -37,6 +37,26 @@ Add plugin to `app.json`. For example:
 }
 ```
 
+## iOS Build Properties
+
+### Experimental Features
+
+#### `ios.buildFromSource` (boolean, iOS only, experimental)
+
+If set to `false`, this enables experimental support for prebuilt React Native iOS dependencies (ReactNativeDependencies.xcframework) by setting `ENV['RCT_USE_RN_DEP'] = '1'` in the Podfile. This can speed up iOS builds and reduce dependency-related build issues. See the [React Native 0.80 release blog post](https://reactnative.dev/blog/2025/06/12/react-native-0.80#experimental---react-native-ios-dependencies-are-now-prebuilt) for more details.
+
+**Example:**
+
+```json
+{
+  "ios": {
+    "buildFromSource": false
+  }
+}
+```
+
+> **Warning:** This feature is experimental and may change or be removed in future releases.
+
```

Comment:
```suggestion
```

i'm leaning toward to remove the doc here. the tsdoc and the generated doc should be good enough in https://docs.expo.dev/versions/unversioned/sdk/build-properties/#pluginconfigtypeios.

adding doc here i guess we will forget to maintain it from time to time

Reply:
Okay, I'm fine keeping just the doc